sql/driver/conn.go

```diff
@@ -40,7 +41,7 @@ func (c *conn) Prepare(query string) (driver.Stmt, error) {
 }
 
 func (c *conn) Begin() (driver.Tx, error) {
-	return &tx{conn: c}, nil
+	return &tx{conn: c, txn: []byte{}}, nil
```

Comment:
I think this should either send a `BEGIN` statement to the server, or queue up a `BEGIN` to be prepended to the next request sent to the server.